packages/ui/src/views/webhooks/webhook-list/repo-webhook-list-page.tsx

```diff
@@ -1,208 +1,48 @@
 import { useEffect, useState } from 'react'
 import { Link, useNavigate } from 'react-router-dom'
 
-import {
-  Filters,
-  FiltersBar,
-  type FilterCondition,
-  type FilterOption,
-  type FilterValue,
-  type SortDirection,
-  type SortOption,
-  type SortValue
-} from '@components/filters'
+import { Filters, FiltersBar } from '@components/filters'
 import useFilters from '@components/filters/use-filters'
 import { Button, ListActions, PaginationComponent, SearchBox, SkeletonList, Spacer, Text } from '@components/index'
 import { useCommonFilter } from '@hooks/useCommonFilter'
+import {
+  getFilterOptions,
+  getLayoutOptions,
+  getSortDirections,
+  getSortOptions
+} from '@views/repo/repo-list/filter-options'
 import { formatDistanceToNow } from 'date-fns'
```

Comment:
same issue with `date-fns` package. this should not be added and neither should it be required.
I know this is existing code, but just making a note here for later.

Reply:
I’ve left a comment in the code to revisit the approach to using `date-fns` in the future and to plan for fully phasing out its usage.